docs/sources/developers/plugins/create-a-grafana-plugin/extend-a-plugin/add-support-for-annotations.md

```diff
@@ -1,16 +1,25 @@
 ---
-title: Add support for annotations
+title: Add annotations
+aliases:
+  - ../../../plugins/add-support-for-annotations/
+description: Add support for annotations in your plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - annotations
+weight: 100
 ---
 
-# Add support for annotations
+# Add annotations
```

Comment:
I'm not sure you're adding annotations, but the ability for users to add annotations, so the former makes more sense to me. An alternative from below could be `Enable annotations`

Reply:
Technically, yes. However, when a shopper goes to buy milk they want milk not the grocery cart that's going to support their ability to buy the milk. :)

Changing to "Enable annotations".